checks.d/powerdns_recursor.py

```diff
@@ -23,24 +25,141 @@ class PowerDNSRecursorCheck(AgentCheck):
         'answers100-1000',
         'cache-hits',
         'cache-misses',
+        'dont-outqueries',
+        'ipv6-outqueries',
+        'ipv6-questions',
         'noerror-answers',
+        'nxdomain-answers',
         'outgoing-timeouts',
+        'over-capacity-drops',
+        'packetcache-hits',
+        'packetcache-misses',
         'questions',
         'servfail-answers',
+        'tcp-client-overflow',
+        'tcp-clients',
         'tcp-outqueries',
         'tcp-questions',
+        'throttle-entries',
+        'throttled-out',
+        'throttled-outqueries',
+        'unauthorized-tcp',
+        'unauthorized-udp',
+        'unexpected-packets',
+    ]
+    GAUGE_METRICS_V4 = [
+        'cache-entries',
+        'concurrent-queries',
+        'failed-host-entries',
+        'negcache-entries',
+        'packetcache-entries',
+        'throttle-entries',
+        'fd-usage',
+    ]
+    RATE_METRICS_V4 = [
```

Comment:
It seems like we have many metrics which are common to V3 and V4... should we maybe create a base set of metrics common to both versions, and then add to the catalog depending on the version? The less duplicates we have the easier it'll be to maintain and less likely we might end up leaving something out. This consolidation is always something we could do in the future, but ideally that would be awesome.